plugin.program.mceremote/changelog.txt

```diff
@@ -0,0 +1,34 @@
+[B]Version 3.0.0 16th March 2021[/B]
```

Comment:
Changelog is no longer displayed in Kodi. You might want to add the most recent changes to the news tag in addon.xml